Form-Controls/index.html

```diff
@@ -16,6 +16,59 @@ <h1>Product Pick</h1>
   <form>
     <!-- write your html here-->
     <!-- try writing out the requirements first-->
+    <fieldset class="validate-account">
+      <label for="fname">First name:</label>
+      <input type="text" id="fname" name="fname" required><br><br>
+      <label for="lname">Last name:</label>
+      <input type="text" id="lname" name="lname" required><br><br>
+      <label for="email">Email:</label>
+      <input type="email" id="email" name="email" required><br><br>
+    </fieldset>
+
+    <section class="select-colour">
+      <label for="color-picker">Choose your t-shirt colour from the three available colours</label>
+      <input type="color" list="presets" id="color-picker">
+      <datalist id="presets">
+        <option value="#FF0000">Red</option>
+        <option value="#FFFF00">Yellow</option>
+        <option value="#6699cc">Blue</option>
+      </datalist>
+    </section>
+
+    <section class="select-size">
+      <div>
```

Comment:
And here you've used a bunch of divs. What are these divs doing in the code? What effect do they have?

Something special happens with fieldset and radio inputs. Can you find out what?